src/GHC/RTS/Events/Incremental.hs

```diff
@@ -209,3 +211,58 @@ mkEventDecoder header = G.runGetIncremental $ getEvent parsers
         , perfParsers
         ]
     parsers = EventParsers $ mkEventTypeParsers imap event_parsers
+
+-- | Writes the 'EventLog' to file. The log is expected to __NOT__ have 'EventBlock'
+-- markers/events - the parsers no longer emit them and they are handled behind
+-- the scenes.
+writeEventLogToFile :: FilePath -> EventLog -> IO ()
+writeEventLogToFile fp = BL.writeFile fp . serialiseEventLog
+
+-- | Serialises an 'EventLog' back to a 'ByteString', usually for writing it
+-- back to a file.
+serialiseEventLog :: EventLog -> BL.ByteString
+serialiseEventLog el@(EventLog _ (Data events)) =
```

Comment:
No, not really because it hasn't changed aside some code formatting and renaming (`isCap` -> `mkCap`).

I agree that it's a good thing to write some tests. I guess we start by fixing #14.